### DISCUSSION TITLE
brew services start mongodb-community

### DISCUSSION THREAD

**Discussion by praney-pareek** [Homebrew/discussions#159] [Everyday usage]
During entering command "brew services start mongodb-community" It showed Python cannot be opened because of a problem.It ask me to check with developer to make sure Python works with this version of macOS. Can you please help me how deal with this problem.

<img width="1440" alt="Screenshot 2020-11-21 at 1 33 56 AM" src="https://user-images.githubusercontent.com/72239881/99846505-730c2380-2b9c-11eb-870b-0ba1d6e85b4f.png">

**Comment by SMillerDev** [comment]
MongoDB isn't distributed by homebrew and you installed python from the official package so I'm not sure there's anything Homebrew can help you with
